BurnerB/Questioner-ADC-
# Bg fix user signin endpoint 163166993

#### What does this PR do?
FIx user sign in endpoint
#### Description of Task to be completed?
Fix uuser sign in ,if password entered match with password in self.db 
the user should get a successfiully signed in message
#### How should this be manually tested?
clone this repo
CD into it
Set up a virtual environment by:
`sudo install virtualenv env`
then `source venv/bin/activate`

install all modules in requirements.txt
run flask by:
`FLASK_APP=run.py`
`FLASK_ENV =development`
`Flask run`
The app should now be running
copy the signin route :
>/api/v1/user/auth/signin

Paste the route in postman and test on HTTP verb POST

#### What are the relevant pivotal tracker stories?
#163166993
#### Screenshots (if appropriate)
![user logged in finally](https://user-images.githubusercontent.com/45785786/51075154-f606ed80-1698-11e9-96e4-49e98fcf540c.png)
